pyomo/contrib/piecewise/piecewise_linear_function.py

```diff
@@ -308,7 +341,13 @@ def _construct_simplices_from_multivariate_points(self, obj, points, dimension):
             # checking the determinant because matrix_rank will by default calculate a
             # tolerance based on the input to account for numerical errors in the
             # SVD computation.
-            if (
+            if tri != Triangulation.Delaunay:
+                # Note: do not sort vertices from OrderedJ1, or it will break.
+                # Non-ordered J1 is already sorted, though it doesn't matter.
+                # Also, we don't need to check for degeneracy with simplices we
+                # made ourselves.
```

Comment:
For future us, it might be better to write this if as `tri in (Triangulation.J1, Triangulation.OrderedJ1)` since it's possible others will need the check?

Reply:
Changed